source/AsyncRgbLedAnalyzerSettings.h

```diff
@@ -10,18 +10,33 @@ class AsyncRgbLedAnalyzerSettings : public AnalyzerSettings
 	AsyncRgbLedAnalyzerSettings();
 	virtual ~AsyncRgbLedAnalyzerSettings();
 
-	virtual bool SetSettingsFromInterfaces();
+	bool SetSettingsFromInterfaces() override;
 	void UpdateInterfacesFromSettings();
-	virtual void LoadSettings( const char* settings );
-	virtual const char* SaveSettings();
+	void LoadSettings( const char* settings ) override;
+	const char* SaveSettings() override;
 
-	
-	Channel mInputChannel;
-	U32 mBitRate;
+	enum Controller {
```

Comment:
We should assign any serialized enums specific integer values.
1) we'll be serializating these settings and then deserializing them on different operating systems.
2) in the future we'll add new LED modules as they are released. It's important that the integer mappings don't change when an older saved file is loaded into a newer version of the analyzer.
If we assign integer values now, I think it it's less likely a future developer might inadvertently insert a new enum value (such as a hypothetical `LED_WS2814`) in the middle of the list, causing the integer mappings to change.

Reply:
Sorry, I normally do this, when hardware is involved, just forgot here, thanks for catching.